src/replication_test.c

```diff
@@ -249,6 +250,17 @@ test_read_data(int fd, uint8_t *data, uint64_t len)
 	return nbytes;
 }
 
+static bool
+is_valid_rebuild_status(zrepl_status_ack_t *zrepl_status) {
+	if (zrepl_status->state == ZVOL_STATUS_DEGRADED)
+		return true;
+	if (replication_factor == 1 &&
```

Comment:
do we have a test case for this scenario in istgt?

Reply:
I think there are no test cases for this scenario. Since at target side, rebuilding is triggered only after the response from opcode `ZVOL_OPCODE_REPLICA_STATUS`. In this, if we get a `Healthy` status rebuild itself will not trigger.